### ISSUE TITLE
Redesign histo-pheno visualization

### ISSUE THREAD

**Issue by vincerubinetti** [monarch-initiative/monarch-app#84]
Related: https://github.com/monarch-initiative/monarch-ui/issues/27

Many folks in the lab have requested/suggested changes to the design of the histo-pheno visualization that appears on disease node pages.

The current design:

<img width="705" alt="image" src="https://user-images.githubusercontent.com/8326331/168663155-f2e12213-18c0-4751-9e7b-6c6c9feb9d22.png">

The requested design/changes:

![image](https://user-images.githubusercontent.com/8326331/168682439-a59fe51c-0cfa-467b-baf5-5c0d1008f8b2.png)

As a result, I've held off on porting over the existing 2.0 histopheno chart to 3.0, because it was just going to change later anyway.

Let's decide on a final design here, making sure that I actually have the needed info from the backend.

**Comment by jmcmurry** [comment]
Thanks Vince. What requested changes are you referring to? There were additional features to bake in ... in the fullness of time, but I think the bare bones version looks and works great. Also not clear on the diff between 2.0 and 3.0

**Comment by vincerubinetti** [comment]
But before I proceed doing it, it seems that there are several questions that still need to be answers in https://github.com/monarch-initiative/monarch-ui/issues/27 .

I wouldn't start coding this until those questions are resolved, as this is more than just baking in more features.

I created this issue just as a way to track the task of implementing histopheno. These questions don't have to be answered right now.
